scripts/Engineering/EnginX.py

```diff
@@ -0,0 +1,381 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2019 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+from __future__ import (absolute_import, division, print_function)
+import mantid.simpleapi as simple
+import Engineering.EnggUtils as Utils
+import os
+import csv
+from shutil import copy2
+
+
+# take the user, the run_number of the vanadium, and the directory to save to as arguments
+def main(vanadium_run, user, focus_run, **kwargs):
+
+    # Set all values at top,
+    # get whether or not to force a vanadium, default value is false
+    do_van = kwargs.get("force_vanadium", False)
+    # get whether or not to pre-process the run before focusing, default is false
+    do_pre_process = kwargs.get("pre_process_run", False)
+    # get whether or not to force re-create the calibration, default is false
+    do_cal = kwargs.get("force_cal", False)
+    cropped = kwargs.get("crop_type", None)
+    ceria_run = kwargs.get("ceria_run", "241391")
+    crop_name = kwargs.get("crop_name", "cropped")
```

Comment:
Can you document each of these with a comment. I want a user to look at it and understand what the variable means

Reply:
No that's ok.